app/ui/omni-box.js

```diff
@@ -33,12 +35,18 @@ class OmniBox extends HTMLElement {
       this.input.select()
     })
 
-    this.form.addEventListener('submit', (e) => {
+    this.form.addEventListener('submit', e => {
```

Comment:
@RangerMauve I ran `yarn lint` before committing so I'm not sure why there are formatting differences.

Reply:
That's okay, did you happen to run prettier on the repo? It might have been from that.

Reply:
I have prettier set up in vscode, so yes. But I ran yarn lint immediately before committing so I don't understand why it's different. Does standard have a config file I need to use?